### PR TITLE
[Csl-878] quizzes library support python

### PR DESCRIPTION
**Pull request by stanlp1**
Catalog test was failing because exception message changed so I also changed it here.

### CODE REVIEW

File: constructor_io/modules/quizzes.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        raise ConstructorException('quiz_id is a required parameter of type str')
+
+    if path == 'finalize' and (not isinstance(parameters.get('a'), list) or len(parameters.get('a')) == 0): # pylint: disable=line-too-long
+        raise ConstructorException('a is a required parameter of type list')
+
+    if parameters:
+        if parameters.get('section'):
+            query_params['section'] = parameters.get('section')
+
+        if parameters.get('version_id'):
```
Comment (jjl014) [inline constructor_io/modules/quizzes.py:32]: I just realized that `version_id` can be added as a parameter, but is this actually used? It's not specified in the [docs](https://docs.constructor.io/rest_api/quiz/using_quizzes/) as a parameter. Might be an artifact of the old docs? 🤔 

I think we allow it to be passed as a parameter in the other SDK repos as well... hm...

**Review by jjl014 — APPROVED** [review]
Good stuffs, this lgtm!

I did leave a question around `version_id` though. 🤔

File: constructor_io/modules/quizzes.py (hunk 1 above)
Comment (stanlp1) [reply constructor_io/modules/quizzes.py:32]: It's specified here. It is supported I guess they just forgot to add it here.

https://dev-quizzes.cnstrc.com/docs/

Comment (jjl014) [reply constructor_io/modules/quizzes.py:32]: Cool beans! 😄

**Pull request merged**